package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lblod/ember-rdfa-editor",
-  "version": "9.14.1",
+  "version": "10.0.2",
```

Comment:
given we're merging into a 9.x branch, I feel this should be here, same with all the changelog mentions of 10.x

Reply:
if you developed this on the master branch and then pr-d to 9.x, you'd have to cherry-pick I think

Reply:
Yeah, it seems I forgot to rebase this PR on the 9.x branch, will do that now

Reply:
should be fixed now.